lib/currency.js

```diff
@@ -1,22 +1,38 @@
-const { Amount, pkg: hsdPkg } = require('hsd');
+const { Amount } = require('hsd');
+
+const pkgs = {
+  handshake: require('hsd/lib/pkg'),
+  bitcoin: require('bcoin/lib/pkg'),
+};
+
 const assert = require('bsert');
 
 const { isChainSupported } = require('./chain');
 
 export const CURRENCY_UNITS = {
   bitcoin: {
-    currency: 'bitcoin',
-    unit: 'btc',
-    milli: 'mbtc',
+    currency: pkgs.bitcoin.currency,
+    unit: pkgs.bitcoin.unit,
+    milli: `m${pkgs.bitcoin.unit}`,
     micro: 'bit',
-    base: 'satoshi',
+    base: pkgs.bitcoin.base,
+    ticker: pkgs.bitcoin.unit.toUpperCase(),
   },
   handshake: {
-    currency: hsdPkg.currency,
-    unit: hsdPkg.unit,
-    milli: `m${hsdPkg.unit}`,
-    micro: `u${hsdPkg.unit}`,
-    base: hsdPkg.base,
+    currency: pkgs.handshake.currency,
+    unit: pkgs.handshake.unit,
+    milli: `m${pkgs.handshake.unit}`,
+    micro: `u${pkgs.handshake.unit}`,
+    base: pkgs.handshake.base,
+    ticker: pkgs.handshake.unit.toUpperCase(),
+  },
+  bitcoincash: {
```

Comment:
I see that bch pkg exists too: https://github.com/bcoin-org/bcash/blob/master/lib/pkg.js

Reply:
will update